### PR TITLE
V1.7.8 patchset

### PR DESCRIPTION
**Pull request by squat**
Carries patches form v1.7.7. Tests are passing.

cc @aaronlevy @s-urbaniak

### CODE REVIEW

File: cluster/images/hyperkube/Makefile [hunk 1]
```diff
@@ -84,6 +84,8 @@ else
 endif
 	# Download CNI
 	curl -sSL --retry 5 https://storage.googleapis.com/kubernetes-release/network-plugins/cni-${ARCH}-${CNI_RELEASE}.tar.gz | tar -xz -C ${TEMP_DIR}/cni-bin
+	curl -sSL --retry 5 -o  ${TEMP_DIR}/cni-bin/bin/calico https://github.com/projectcalico/calico-cni/releases/download/v1.9.1/calico
```
Comment (s-urbaniak) [inline cluster/images/hyperkube/Makefile]: there is v1.11.0 out https://github.com/projectcalico/calico-cni/releases.

please ignore me if this is not relevant.

**Review by s-urbaniak — APPROVED** [review]
LGTM modulo a small question regarding cni-plugin.

**Comment by squat** [comment]
@s-urbaniak hmm yeah good call. Bumping to v1.11.0. Re-running tests then merging.

**Comment by squat** [comment]
Tests passed; merging.

**Pull request merged**